dot/telemetry/mailer_test.go

```diff
@@ -145,7 +148,9 @@ func TestListenerConcurrency(t *testing.T) {
 	for i := 0; i < qty; i++ {
 		go func() {
 			bestHash := common.Hash{}
-			GetInstance().SendMessage(NewBlockImportTM(&bestHash, big.NewInt(2), "NetworkInitialSync"))
+			//GetInstance().SendMessage(NewBlockImportTM(&bestHash, big.NewInt(2), "NetworkInitialSync"))
+			msg := NewBlockImportTM(&bestHash, big.NewInt(2), "NetworkInitialSync")
+			SendMessage(msg)
```

Comment:
A bit out of scope, so feel free to ignore. Just some knowledge sharing!

That could be improved such that:

- all goroutines start when they are all ready
- they all run for a fixed mount of time i.e. 50ms

Example I used to test logger data races 

https://github.com/ChainSafe/gossamer/blob/dc8bbefe8acc439be5f08491d8341daa9f63812f/internal/log/race_test.go#L31-L64